controllers/mysqlcluster_controller.go

```diff
@@ -898,6 +898,32 @@ func (r *MySQLClusterReconciler) reconcileV1StatefulSet(ctx context.Context, req
 	if podSpec.SecurityContext.FSGroupChangePolicy == nil {
 		podSpec.SecurityContext.WithFSGroupChangePolicy(corev1.FSGroupChangeOnRootMismatch)
 	}
+	if podSpec.Affinity == nil {
+		podSpec.WithAffinity(corev1ac.Affinity().
+			WithPodAntiAffinity(corev1ac.PodAntiAffinity().
+				WithPreferredDuringSchedulingIgnoredDuringExecution(corev1ac.WeightedPodAffinityTerm().
+					WithWeight(100).
+					WithPodAffinityTerm(corev1ac.PodAffinityTerm().
+						WithLabelSelector(metav1ac.LabelSelector().
+							WithMatchExpressions(
+								metav1ac.LabelSelectorRequirement().
+									WithKey(constants.LabelAppName).
+									WithOperator(metav1.LabelSelectorOpIn).
+									WithValues(constants.AppNameMySQL),
+							).
+							WithMatchExpressions(
```

Comment:
Could you also add a selector for `app.kubernetes.io/created-by: moco`?
Since it is set to backup job. It is better to match them.
https://github.com/cybozu-go/moco/blob/4658a9752e65c256e3c0289cba211102f52e3e11/controllers/mysqlcluster_controller.go#L1254-L1259

Reply:
Thanks, I added:
https://github.com/cybozu-go/moco/pull/513/commits/5e53cb14fa6959bca38f4ec9bfd98e29cf3fff20

Reply:
Thank you. Please update `usage.md` too.

Reply:
added: https://github.com/cybozu-go/moco/pull/513/commits/16aa68e14106553f86365a533ac2a0f15f983e4a